src/navigation/planners/planners/node_ordered_mid_spline.py

```diff
@@ -194,15 +194,39 @@ def map_callback(self, track_msg: ConeDetectionStamped):
             mid_x, mid_y = midpoint([yx[i], yy[i]], [bx[i], by[i]])
             tx.append(mid_x)
             ty.append(mid_y)
+            # for old QUTMS path curve intensity calcs
             th.append(angle([bx[i], by[i]], [yx[i], yy[i]]))  # angle of tangent at midpoint
```

Comment:
While you are here would be good to use the /driverless_common/common.py implementation of angle().